docs/new_flux_ideas_and_guide.md

```diff
@@ -0,0 +1,985 @@
+of. Events are what are called irregular time series given that they can occur at any time. It is possible to create 
+regular time series from an underlying event stream (like counting how many occurred every five minutes for the last hour).
+
+### Working with data in Flux
+When working with data in Flux, it translates the line protocol/time series structure into to a table. You 
+can think of a measurement name as a table with the tag and field keys representing columns. Flux also 
+includes the reserved column, `time`, in every table pulled from InfluxDB. The `time` column is the time at the end 
+of the line protocol when data is written in.
+
+Here's an example of line protocol and its translation to a Flux table:
+
+```
+h2o_temperature,location=santa_monica,state=CA surface_degrees=65.2,bottom_degrees=50.4 1568756160
+h2o_temperature,location=santa_monica,state=CA surface_degrees=63.6,bottom_degrees=49.2 1600756160
+h2o_temperature,location=coyote_creek,state=CA surface_degrees=55.1,bottom_degrees=51.3 1568756160
+h2o_temperature,location=coyote_creek,state=CA surface_degrees=50.2,bottom_degrees=50.9 1600756160
+h2o_temperature,location=puget_sound,state=WA surface_degrees=55.8,bottom_degrees=40.2 1568756160
+h2o_temperature,location=puget_sound,state=WA surface_degrees=54.7,bottom_degrees=40.1 1600756160
+```
+
+| location | state | surface_degrees | bottom_degrees | time |
+| --- | --- | --- | --- | --- |
+| coyote_creek | CA | 55.1 | 51.3 | 1568756160 |
+| coyote_creek | CA | 50.2 | 50.9 | 1600756160 |
+| puget_sound | WA | 55.8 | 40.2  | 1568756160 |
+| puget_sound | WA | 54.7 | 40.1  | 1600756160 |
+| santa_monica | CA | 65.2 | 50.4 | 1568756160 |
+| santa_monica | CA | 63.6 | 49.2 | 1600756160 |
+
+Note that the measurement name doesn't appear anywhere. That's because the table itself represents data from the 
+`h2o_temperature` measurement. Later we'll see how to work with multiple measurements (i.e. tables). Note that the 
+underlying time series would be represented by the combination of `location`, `state`, and the specific field. 
+Results are ordered by default by the columns of the tags and their values and then by time. So we can see that 
+`coyote_creek`, `CA` rows are together in time ascending order.
+
+We'll show later how to specify to the graphing engine what represents an individual time series that you want 
+visualized.
+
+## Basic Queries
+This section will explore some basic queries in Flux to highlight the syntax and a few of the basic concepts of
+the language. For these examples, we'll assume a dataset that has a schema like the one seen above.
+
+### Introduction to basics and getting the last value
+In this example, we'll get the last value of `surface_degrees` for every series in the `h2o_temperature` 
+measurement in the `my_data` bucket. Flux has a shorthand syntax for the common operation of selecting data from 
```

Comment:
Is it important to call this "shorthand" syntax?

Reply:
If this is a special type of syntax, then yeah maybe its helpful to name it for discussion.